onnxruntime/core/providers/nnapi/nnapi_builtin/builders/helper.cc

```diff
@@ -333,6 +333,35 @@ common::Status GetQuantizationZeroPoint(const InitializedTensorSet& initializers
   return Status::OK();
 }
 
+bool IsNodeInQDQGroup(const Node& node) {
+  return target_node_to_qdq_group.find(&node) != target_node_to_qdq_group.end();
+}
+
+std::optional<QDQ::NodeGroup> GetQDQNodeGroup(const onnxruntime::GraphViewer& graph_viewer, const Node& node) {
+  QDQ::ConvSelector qdq_conv_selector;
```

Comment:
I used QDQ::ConvSelector but not the QDQ::BaseSelector here for now. For single conv op case, it's fine.  Because it seems like BaseSelector can not be instantiated as it is an abstract class and has a virtual Check() method needed to be implemented by the inherited classes. 

Not sure how to instantiate it correctly. should be a small problem though.)

Reply:
maybe have a `const QDQ::BaseSelector&` parameter and instantiate the desired selector at the call site?
looks like there will need to be some way to choose the right selector

Reply:
I'd expect something similar to the CPU EP QDQ optimizer is needed where you register selectors against operator types and opset versions. If a match is found the BaseSelector interface is used to do node selection.